04-onion-routing.md

```diff
@@ -598,178 +598,142 @@ this protocol, e.g. via association with an HTLC in a payment channel.
 
 ## Failure Messages
 
-The failure message encapsulated in `failuremsg` has identical format to
-a normal message: 2-byte type (`failure_code`) followed by data suitable
-for that type. The following `failure_code` values are supported.
-A node MUST select one of these codes when creating an error message, and MUST include the appropriate data.
-Notice that the `failure_code` are not message types defined in other BOLTs, not being sent directly on the transport layer, but wrapped inside an return packet.
-The numeric values for the `failure_code` may therefore reuse values that are also assigned as message types, without causing collisions.
+The failure message encapsulated in `failuremsg` has an identical format as
```

Comment:
"identical format to a normal message" or "same format as a normal message".

Reply:
fixed in future commit, see line 642